src/sagemaker_containers/environment.py

```diff
@@ -37,6 +40,7 @@
 BASE_PATH_ENV = 'BASE_PATH'  # type: str
 CURRENT_HOST_ENV = 'CURRENT_HOST'  # type: str
 JOB_NAME_ENV = 'JOB_NAME'  # type: str
+USE_NGINX_ENV = 'SAGEMAKER_USE_NGINX'  # type: str
```

Comment:
? this is already defined below. Seems like the merge was not clean.

Reply:
I will fix it.